### DISCUSSION TITLE
gauge taiko framework run command

### DISCUSSION THREAD

**Discussion by Pekshika** [getgauge/taiko#2741] [Development]
gauge run specs
this command stops if one spec failed, how i can run all specs even if one failed

**Comment by Pekshika** [comment]
using gauge taiko .js

**Comment by zabil** [comment]
There is documentation about this it at

https://docs.gauge.org/writing-specifications.html?os=macos&language=javascript&ide=vscode#continue-on-failure

and 

https://docs.gauge.org/faqs.html?os=macos&language=javascript&ide=vscode#why-we-cannot-skip-all-tests-dynamically-during-a-gauge-run-if-there-is-a-test-failure
